code/controlconfig/controlsconfig.cpp

```diff
@@ -393,21 +393,10 @@ int control_config_detect_axis()
 
 void control_config_conflict_check()
 {
-	int i, j, shift = -1, alt = -1;
-
+	int i, j;
+	
 	for (i=0; i<CCFG_MAX; i++) {
 		Conflicts[i].key = Conflicts[i].joy = -1;
-		switch (Control_config[i].key_id) {
-			case KEY_LSHIFT:
-			case KEY_RSHIFT:
-				shift = i;
```

Comment:
Also, it would appear these variables were missed for removal by Echelon9 in 1a98990c in 2010.  This seems like it would have to be an SVN commit but I can't find the associated revision in Git.  However browsing through SVN by date showed that it was commit r6863.  So it does look like it should have been removed them, although we could let Echelon9 double check his work :)